### PR TITLE
To Do List structure

### PR DESCRIPTION
**Pull request by maccony202**
A project designed  to show a To Do List structure dynamically

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
Hi @maccony202, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you don't have a README template file so kindly implement it with the given template and remeber that Microverse updated it to a new template check this link [NEW README](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.new-task {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 20px;
+}
+
+#newtask {
+  border: hidden;
```
Comment (fobadara) [inline src/style.css:29]: [REQUIRED] Kindly rename this file to `index.css` for the sake of this project.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @maccony202, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._
[REQUIRED] kindly create a modules folder with other .js files as instructed.
[REQUIRED] Your app is not adding and removing books
[REQUIRED] You are not using the Luxon library to display the date and time.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @maccony202, 

# STATUS: APPROVED 😃

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 
- Good implementation of Webpack 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by maccony202** [comment]
> Hi @maccony202,
> 
> Good job so far! You still need to work on some issues to go to the next project, but you are almost there!
> 
> ## Highlights
> ✅ Linters are passing
> 
> ## Required Changes ♻️
> _Check the comments under the review._ [REQUIRED] kindly create a modules folder with other .js files as instructed. [REQUIRED] Your app is not adding and removing books [REQUIRED] You are not using the Luxon library to display the date and time.
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sorry boss i submitted a wrong pull

**Pull request merged**